linting/.core.js

```diff
@@ -0,0 +1,178 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es6": true,
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/eslint-recommended",
+        "plugin:@typescript-eslint/recommended",
+    ],
+    "ignorePatterns": ["*.js", "scripts/**"],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "project": "./tsconfig.base.json",
+        "ecmaFeatures": {
+            "jsx": true,
+        },
+    },
+    "plugins": [
+        "@typescript-eslint",
+    ],
+    "rules": {
```

Comment:
Maybe split this further into `.core.` and `.ts` for `@typescript-eslint specific` ones?